api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-18.xml

```diff
@@ -1186,28 +1188,32 @@
   <concept_reference_source concept_source_id="11" name="PIH" description="Equivalent concept ID in Master Partners-in-Health Concept dictionary" creator="1" date_created="2010-08-03 01:50:49.0" retired="false"  uuid="fb9aaaf1-65e2-4c18-b53c-16b575f2f385"/>
-  <concept_reference_source concept_source_id="12" name="PIH Malawi" description="Partners in Health Malawi concept dictionary" creator="1" date_created="2010-10-24 17:02:20.0" retired="false"  uuid="2b3c2778-768a-102f-83f4-12313b04a615"/>
-  <concept_reference_source concept_source_id="13" name="AMPATH" description="AMPATH concept dictionary" creator="1" date_created="2010-10-24 17:02:38.0" retired="false"  uuid="2b3c28ae-768a-102f-83f4-12313b04a615"/>
-  <concept_reference_source concept_source_id="14" name="SNOMED MVP" description="MVP Namespace Identifier extensions to SNOMED CT" creator="1" date_created="2010-12-03 18:18:39.0" retired="false"  uuid="2b3c2a2a-768a-102f-83f4-12313b04a615"/>
+  <concept_reference_source concept_source_id="12" name="PIH Malawi" description="Partners in Health Malawi concept dictionary" creator="1" date_created="2010-10-24 17:02:20.0" retired="false"  uuid="12ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"/>
+  <concept_reference_source concept_source_id="13" name="AMPATH" description="AMPATH concept dictionary" creator="1" date_created="2010-10-24 17:02:38.0" retired="false"  uuid="13ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"/>
+  <concept_reference_source concept_source_id="14" name="SNOMED MVP" description="MVP Namespace Identifier extensions to SNOMED CT" creator="1" date_created="2010-12-03 18:18:39.0" retired="false"  uuid="14ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"/>
   <concept_reference_source concept_source_id="15" name="org.openmrs.module.mdrtb" description="The required concepts for the MDR-TB module" creator="1" date_created="2010-12-15 14:00:37.0" retired="false"  uuid="ddb6b595-0b85-4a80-9243-efe4ba404eef"/>
-  <concept_reference_source concept_source_id="18" name="HL7 2.x Route of Administration" description="Members of value set from HL7 for routes of administration" hl7_code="HL70162" creator="1" date_created="2012-10-19 13:43:57.0" retired="false"  uuid="98e77229-19e7-45e0-88dd-e30214dd2fad"/>
-  <concept_reference_source concept_source_id="19" name="3BT" description="Belgian Bilingual Bi-encoded Thesaurus (3BT)" creator="1" date_created="2013-02-16 22:25:25.0" retired="false"  uuid="ef7f179f-d97d-4a8c-994c-67c544428684"/>
-  <concept_reference_source concept_source_id="20" name="ICPC2" description="International Classification of Primary Care version 2 from WONCA." creator="1" date_created="2013-02-17 15:25:41.0" retired="false"  uuid="32fb691d-928b-48a9-b3ee-ca64f21f20bf"/>
-  <concept_reference_source concept_source_id="21" name="CIEL" description="Columbia International eHealth Laboratory concept ID" creator="1" date_created="2013-07-03 15:55:03.0" retired="false"  uuid="249b13c8-72fa-4b96-8d3d-b200efed985e"/>
```

Comment:
What has actually changed in these lines?

Reply:
@dkayiwa , is it really necessary to manually revert all the changes? I have already changed so many dates. All of them have been modified by the data exchange module. Doesn't the amount of manual effort defeat the purpose of the module?

Reply:
Does this mean that the dates had changed from the last time an export was done?

Reply:
@dkayiwa , here I guess the major change is the modification of the UUID and not the dates.